src/services/UpdateTechRecordService.ts

```diff
@@ -16,7 +16,7 @@ export class UpdateTechRecordService {
                 newStatus,
             },
             httpMethod: HTTPMethods.PUT,
-            resource: "/vehicles/update-status/{vin}"
+            resource: "/vehicles/update-status/{sysNum}"
```

Comment:
I need to test this to confirm but I think this parameter should be called `systemNumber` to be correctly retrieved from the lambda event

Reply:
Ok, the resource field doesn't influence the call. Resolving